src/main/java/se/sundsvall/billingpreprocessor/api/BillingRecordsResource.java

```diff
@@ -61,6 +66,17 @@ public ResponseEntity<Void> createBillingRecord(final UriComponentsBuilder uriCo
 		return ResponseEntity.created(uriComponentsBuilder.path("/billingrecords/{id}").buildAndExpand(uuid).toUri()).header(CONTENT_TYPE, ALL_VALUE).build();
 	}
 
+    @PostMapping(path = "/batch", consumes = APPLICATION_JSON_VALUE, produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
+    @Operation(summary = "Create billing records", description = "Creates new billing records defined by the supplied attributes")
+    @ApiResponse(responseCode = "201", headers = @Header(name = LOCATION, schema = @Schema(type = "string")), description = "Successful operation", content = @Content(mediaType = ALL_VALUE, schema = @Schema(implementation = Void.class)))
+    @ApiResponse(responseCode = "400", description = "Bad request", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(oneOf = {Problem.class, ConstraintViolationProblem.class})))
+    @ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+    public ResponseEntity<List<String>> createBillingRecords(final UriComponentsBuilder uriComponentsBuilder, @Valid @NotNull @RequestBody final List<BillingRecord> billingRecords) {
+
+        var uuid = service.createBillingRecords(billingRecords);
```

Comment:
uuid -> uuidList ?

(since the uuid contains a List of strings, this name change makes it easier to read the code, i.m.h.o.)